INSTALL.md

```diff
@@ -140,19 +140,51 @@ pecl install oci8-2.0.10
 ```
 This will download and compile the module, and place it in your php extension dir.
 
-Add the ```extension=oci8.so``` line to your php.ini or create a configuration file: 
+Add the ```extension=oci8.so``` line to your php.ini or create a configuration file:
 
-```bash 
+```bash
 echo 'extension=oci8.so' > /etc/php.d/oci8.ini
 ```
 
 Confirm it is working with ```php -i | grep -i oci8```
 
+#### MariaDB/mysql
+The following instructions are to set up a local MYSQL database for GOCDB. They have not been tested in a production enviroment and are currently intended for test instances.
+
+First you will need to install MariaDB. Then start the 'mysqld' service.
+
+Access the CLI for MariaDB
```

Comment:
as this section is MariaDB and MySQL instructions (not just MariaDB), should this read something like: 

> Access the Database CLI

Reply:
The line about MariaDB installation should be generalised as well then.

Reply:
references to MariaDB or MySQL have been generalized throughout